### DISCUSSION TITLE
How to pack latest changes using npm pack?

### DISCUSSION THREAD

**Discussion by gschauhan** [npm/feedback#514] [CLI] [answered]
I am trying to test one of the package locally. What I do is go the root directory of my package and run

npm pack

It creates .tgz file and I install this file via npm i <path to file>. Now I found that some of my recent changes are missing in this .tgz file. I have made some tiny modifications in css file and those are not reflected in packaged .tgz file. I am not sure what is wrong here. I have looked into internet but all questions in SA and other resources says to use 'npm pack' to create tgz file, but not about this issue.

npm version 6.14.13

**Comment by MylesBorins** [comment, marked as answer]
hey @gschauhan. `npm pack` uses the files on the local file system to create the package .tgz file. It is possible that when running `npm install` you are getting a cached version on this system if the version is the same. You might want to try bumping the version number with an rc or alpha in the semver.

With that said, this discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!

**Reply by gschauhan** [reply]
I was struggling with this issue so posted here. Thanks so much for reply. I already had Stackover flow question but didnt receive any reply. Will confirm this and post reply there so people can find it.
